apps/web/views/pets/update.rb

```diff
@@ -0,0 +1,7 @@
+module Web::Views::Pets
```

Comment:
Copy from [lotus/view readme](https://github.com/lotus/view).

"It's based on a separation between views and templates.

A view is an object that encapsulates the presentation logic of a page. A template is a file that defines the semantic and visual elements of a page. In order to show a result to a user, a template must be rendered by a view.

Keeping things separated helps to declutter templates and models from presentation logic. Also, since views are objects, they are easily testable. If you ever used Mustache, you are already aware of the advantages.

Like all the other Lotus components, it can be used as a standalone framework or within a full Lotus application."